packages/components-js/tests/vrt/puppeteer/helpers/puppeteer-helper.ts

```diff
@@ -13,6 +14,15 @@ export const setContentWithDesignSystem = async (page: Page, content: string, op
     ...opts,
   };
 
+  const initialStyles = getInitialStyles();
+  // Unsupported media feature: hover
+  const initialStylesWithoutMediaQuery = initialStyles
+    .replace(/\@media\(hover\:hover\)\{/g, '')
+    .replace(
```

Comment:
Not sure why we would need the 2nd replacement.

Reply:
added sub task